app/src/components/Market/OrderCard/components/BuyOrMint/components/Buy.tsx

```diff
@@ -2,16 +2,16 @@ import React, { useCallback, useState } from 'react'
 
 import { useWeb3React } from '@web3-react/core'
 
-import Box from 'components/Box'
-import Button from 'components/Button'
-import Input from 'components/Input'
-import Label from 'components/Label'
-import Spacer from 'components/Spacer'
+import Box from '@/components/Box'
+import Button from '@/components/Button'
+import Input from '@/components/Input'
+import Label from '@/components/Label'
+import Spacer from '@/components/Spacer'
```

Comment:
Oh I see, this is how next can make the imports global?

Reply:
Yep thats how absolute imports work in next.js

Reply:
I didn't replace all the absolute imports but whenever I came across them in a build error I replaced them.  In production I will only use them.